src/components/AddElement.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
+
+import { AddCircleIcon } from '../../assets/images'
+import { ContentSecondary } from './text/Content'
+import { SubheadingPrimary } from './text/Subheading'
+
+const Root = styled.TouchableOpacity`
+  margin: ${props => props.theme.spacings.sm} 0;
+`
+
+const AddCustomDisciplineText = styled(SubheadingPrimary)`
+  text-transform: uppercase;
+  padding-left: ${props => props.theme.spacings.xs};
+`
+
+const Explanation = styled(ContentSecondary)`
+  padding: 0 ${props => props.theme.spacings.xxl} 0 11%;
```

Comment:
can't we use either xl(10%) or xxl(12%) ? even if the design differs we should use defined spacings

Reply:
this padding needs to be the same as the width of the icon plus the padding of the text, so these text is exactly beneath the other one. For better understanding have a look at the designs, which makes it clear.

Reply:
alright keep it like that.